apps/src/netsim/DashboardUser.js

```diff
@@ -91,7 +91,9 @@ DashboardUser.getCurrentUser = function () {
         DashboardUser.currentUser_.initialize_(data);
       },
       error: function (/*jqXHR, textStatus, errorThrown*/) {
-        throw new Error("Unable to retrieve current user info.");
+        DashboardUser.currentUser_.initialize_({
```

Comment:
Looking at this function further, is the comment accurate - it doesn't look like we give it a callback. It just gets called and sets currentUser_? Will it be bad if someone else tries to use DashboardUser.currentUser_ before hearing back from the ajax call?